src/EnergyPlus/SolarShading.cc

```diff
@@ -5984,127 +5987,128 @@ namespace SolarShading {
 						// Shade or screen or blind on, or switchable glazing
 						// (note in the following that diffusing glass is not allowed in a window with
 						// shade, blind or switchable glazing)
+						if ( ConstrNumSh != 0 ) {
+							if ( ShadeFlag != IntBlindOn && ShadeFlag != ExtBlindOn && ShadeFlag != BGBlindOn && ShadeFlag != ExtScreenOn ) {
 
-						if ( ShadeFlag != IntBlindOn && ShadeFlag != ExtBlindOn && ShadeFlag != BGBlindOn && ShadeFlag != ExtScreenOn ) {
-
-							// Shade on or switchable glazing
+								// Shade on or switchable glazing
 
-							if ( SunLitFract > 0.0 ) TBmAllShBlSc = POLYF( CosInc, Construct( ConstrNumSh ).TransSolBeamCoef );
+								if ( SunLitFract > 0.0 ) TBmAllShBlSc = POLYF( CosInc, Construct( ConstrNumSh ).TransSolBeamCoef );
```

Comment:
Stray whitespace

Reply:
There is "if" statement in line 5990 which goes all the way to 6111. That is why additional additional indentation is here. (Unless I am missing something else?)